pages/docs/create.md

```diff
@@ -107,8 +107,8 @@ db.Model(&User{}).Create(map[string]interface{}{
   "Name": "jinzhu", "Age": 18,
 })
 
-// batch insert from `[]map[string]interface{}{}`
-db.Model(&User{}).Create([]map[string]interface{}{
+// batch insert from `&[]map[string]interface{}{}`
+db.Model(&User{}).Create(&[]map[string]interface{}{
```

Comment:
I found that if I use the `sqlite` driver, then the reference of the map is required here. However, if I use the `mysql` driver, then it's not needed here. Interesting thing.